modules/disko/disko-ab-partitions.nix

```diff
@@ -91,18 +92,32 @@
                   #randomEncryption = true;
                 };
               };
-              zfs_1 = {
-                size = "100%";
+              zfs_root = {
+                size = "30G";
                 content = {
                   type = "zfs";
-                  pool = "zfspool";
+                  pool = "zfs_root";
+                };
+              };
+              zfs_data = {
```

Comment:
If you are still using this post boot script, than this should be just an empty partition.

Reply:
If using post boot script, we keep empty partition and we do something similar to [this](https://github.com/tiiuae/ghaf/pull/517/commits/b11ed1ca92558cff00d57379f4a65c4a52341db8#diff-be3c6878b82afb556d2957d8b8bbd53f6bd0d3b49475f65ccd9ba891cd9c5480R28-R31)?

Reply:
Yes. We need to detect if the device already has luks headers and if only format if this is not the case.

Reply:
Addressed, now we have better control on password with `post boot script`.
My only concern is once I use `luksFormat` on `zfs_data` partition all zfs meta data (pools, datasets) info is lost, because of that I need to create pool and datasets again in `post boot script`.